src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -919,11 +919,8 @@ void FlexPA::prepPoint_pin_checkPoint_planar(
   design_rule_checker.main();
   design_rule_checker.end();
 
-  if (design_rule_checker.getMarkers().empty()) {
-    ap->setAccess(dir, true);
-  } else {
-    ap->setAccess(dir, false);
-  }
+  const bool no_drv = design_rule_checker.getMarkers().empty();
```

Comment:
When did gc_worker change to design_rule_checker? I am ok with changing the classname but the object should follow the class name untill this is done.

Reply:
PR [#5760](https://github.com/The-OpenROAD-Project/OpenROAD/pull/5760)
The reasoning is that in this case the name "design_rule_checker" makes it clear what this object is doing even if you are not familiar with the code base.

Reply:
In future changes, I prefer keeping the name of the object matching the name of the class until the class name is changed.